File: front/src/app/core/domain/message/start-game.message.ts
import {BaseMessage} from "@domain/message/base.message";
import {MessageTypeEnum} from "@domain/message/message-type.enum";
import {QuestionDto} from "@domain/questions/question.dto";

export class StartGameMessage extends BaseMessage {
    override readonly type = MessageTypeEnum.START_GAME;
    question: QuestionDto;
}

File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/StartGameMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.question.QuestionDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StartGameMessage extends BaseMessage {
    private QuestionDto question;

    public StartGameMessage() {
        super(MessageType.START_GAME);
    }
}
